board/main.c

```diff
@@ -628,6 +630,15 @@ int usb_cb_control_msg(USB_Setup_TypeDef *setup, uint8_t *resp, bool hardwired)
         heartbeat_counter = 0U;
         break;
       }
+    // **** 0xf4: k-line/l-line 5 baud initialization
+    case 0xf4:
+      k_wakeup = (setup->b.wValue.w == 0U) || (setup->b.wValue.w == 2U);
```

Comment:
Would maybe add a check here to make sure we are on a white/grey panda. On black/uno, GPIOC12 is used for other purposes, so that might screw with it